src/components/BucketsList.tsx

```diff
@@ -0,0 +1,62 @@
+import { memo } from 'react'
+import { Link } from 'react-router-dom';
+import Utils from '../utils';
+import BucketTypeIcon from './BucketTypeIcon';
+
+type Bucket = {
+  bucket: string;
+  manifests: number;
+  official: boolean;
+};
+
+function BucketsList({ results, keyword }) {
+    const keywordRE = new RegExp(`${keyword}`, 'g');  
+
+    const setKeywordHighlight = function(bucketName: string) {
+        if (!keyword) {
+            return bucketName;
+        }
+
+        const ret = bucketName.replace(keywordRE, `<mark style="padding: 0;">${keyword}</mark>`);
+        return ret;
+    }
+
+    const filterIncludesKeywordBuckets = (item: Bucket) => {
```

Comment:
Could we update the search query and rely on search results instead of filtering locally the buckets?
The goal is to have a consistent experience between apps and buckets (updated buckets counter for example) and to easily extend the buckets page later (filtering by official/popular buckets...)

Reply:
Okay, I understand🙂